verification/client/abi.go

```diff
@@ -333,7 +394,9 @@ func getProfile(t Transport) (*GetProfileResp, error) {
 		Flags        uint32
 	}{}
 
-	respHdr, err := execCommand(t, CommandGetProfile, 0, cmd, &respStruct)
+	// GetProfile command code is 1 in all revisions of the spec
+	getProfile := CommandCode(0x1)
```

Comment:
Even though it is 0x1 in all versions of the spec, shouldn't we do c.constants.Codes.GetProfile for clarity?

Reply:
The problem is that this function is used to get to profile and then figure out which type of DPEABI to construct. So until we call this command we don't know which profileInfo to use.

Reply:
Ah right that makes sense.